addons/project/controllers/portal.py

```diff
@@ -34,6 +34,9 @@ def _project_get_page_view_values(self, project, access_token, page=1, date_begi
         # pager
         url = "/my/projects/%s" % project.id
         values = self._prepare_tasks_values(page, date_begin, date_end, sortby, search, search_in, groupby, url, domain, su=bool(access_token))
+        # adding the access_token to the pager's url args,
+        # so we are not prompted for loging when switching pages
+        values['pager']['url_args']['access_token'] = access_token
```

Comment:
We could maybe check if the access_token exists since that method could also be used in another route. 🙂

Reply:
I've just tested, if the `access_token` is `None`, it's just not added as an argument of the URL. So an existence condition isn't necessary. I've pushed an additional comment about that :)